bin/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+echo "Installing PHP packages..."
+composer update
+
+echo "Running Polylang build..."
+npm install && npm run build
+
+echo "Build done!"
+
+echo "Cleanup" # Discard composer.lock and package-lock.json changings to ensure they'll never be pushed on repository.
```

Comment:
These lines looks useless to me since `composer.lock` and `package-lock.json` are ignored in `.gitignore` file.
Also, when the script runs, we get the following error:
```
error: pathspec 'composer.lock' did not match any file(s) known to git
error: pathspec 'package-lock.json' did not match any file(s) known to git
```

Reply:
Done in 4184ddaaf0945547f2bb2a5e714ec24cdfdd0a4d